src/halo/conf/cluster.yaml.sample

```diff
@@ -0,0 +1,64 @@
+cortx:
+  external:
+    consul:
```

Comment:
Remove anything that is not required like Consul. Instead Remove everything and include only the bare min things.

Reply:
removed most of the things, for now consul will be required as provisioner reads consul endpoint for copying gconf to consul. we have some backward compatibility code foe consul in provisioner.